packages/jest-cli/src/init/index.ts

```diff
@@ -34,9 +33,17 @@ type PromptsResults = {
 
 const getConfigFilename = (ext: string) => JEST_CONFIG_BASE_NAME + ext;
 
-export default async (
-  rootDir: string = realpath(process.cwd()),
-): Promise<void> => {
+let cwd = process.cwd();
+
+try {
+  cwd = fs.realpathSync.native(cwd);
+} catch (error) {
```

Comment:
can we use `tryRealpath` instead?

Reply:
or create a second helper just for jest-cli, since the other one is in jest-config